src/Altinn.App.Api/Controllers/EventsReceiverController.cs

```diff
@@ -62,11 +69,11 @@ public async Task<ActionResult> Post([FromQuery] string code, [FromBody] CloudEv
             }
             catch (NotImplementedException)
             {
-                return BadRequest();
+                return BadRequest($"No eventhandler found that supports {cloudEvent.Type}");
             }
             catch (Exception ex)
```

Comment:
Catching and handling all exceptions in controllers makes sense to me, should we dismiss the CodeQL alert with won't fix?

Reply:
Yes, in this particular case I wanted a more explicit logging that we would get with the generic error handling which also would return a 500 but with less info logged.